src/components/List/index.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import Card from '../Card/index.js';
+import './list.css';
+
+export defualt class List extends React.Component() {
```

Comment:
in fact, this would be better done using a pure stateless component. this is fine, though